service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceQueries.kt

```diff
@@ -296,12 +296,14 @@ fun Database.Read.fetchChildrenAttendances(unitId: DaycareId, date: LocalDate):
             ca.arrived,
             ca.departed
         FROM child_attendance ca
-        WHERE (ca.departed IS NULL OR ca.departed::date = :date) AND ca.child_id IN ($placedChildrenSql)
+        WHERE ca.child_id IN ($placedChildrenSql)
+        AND (ca.arrived::date = :date OR tstzrange(ca.arrived, ca.departed) @> :departedThreshold OR (ca.arrived::date = ca.departed::date AND ca.arrived::date = :date))
```

Comment:
`ca.arrived::date = :date` is listed in the first and third ORed condition, so the third condition seems to be unnecessary